tests/samples/test_api.py

```diff
@@ -385,8 +385,15 @@ async def test(
 
         assert upload.reserved is True
 
+    @pytest.mark.parametrize("path", ["/samples", "/space/test_id/samples"])
```

Comment:
Space IDs are integers. The path prefix is 'spaces', not 'space'.
```suggestion
    @pytest.mark.parametrize("path", ["/samples", "/spaces/0/samples"])
```